index.html

```diff
@@ -0,0 +1,754 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Space Ripples Reveal Big Bang’s
+          Smoking Gun</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css">
+    </head>
+    <body>
+      <nav class="navbar">
+        <div class="left-logos">
+          <a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a>
+          <a href="#"><i class="fa fa-search" aria-hidden="true"></i></a>
+        </div>
+        <div class="main-logo">
+          <a href="#"><img src="images/the-new-york-times-logo.png" alt="The New York Times logo"></a>
+        </div>
+        <div class="right-buttons">
+          <button type="submit" class="navbar-buttons"><strong>SUBSCRIBE NOW</strong></button>
+          <button type="submit" class="navbar-buttons"><strong>LOG IN</strong></button>
+        </div>
+      </nav>
+      <div class="container">
+
+        <!-- Main Section -->
+
+        <section class="main-section">
```

Comment:
- [ ] **[OPTIONAL]** For good HTML semantics, try to use the `<main>` tag instead of the `<section>`  here